server/frontend/src/containers/App.js

```diff
@@ -20,9 +39,21 @@ class App extends Component {
         <p className="App-intro">
           To get started, edit <code>src/App.js</code> and save to reload.
         </p>
+                <button
+                        onClick={this.keyPressEvent}>
```

Comment:
Try to configure your text editor to use 2 spaces for tabs since it'll cause some inconsistencies.